sdk/js/src/Request.ts

```diff
@@ -6,13 +6,12 @@ import axiosRetry from 'axios-retry'
 import { BucketedUserConfig, DVCLogger } from '@devcycle/types'
 
 const axiosClient = axios.create({
+    timeout: 60 * 1000,
```

Comment:
isn't 60 seconds way too long of a timeout?

Reply:
I set it to 60 because I think that's the timeout of our ALBs. We need some timeout set, so that the exponential backoff stops. By default, there is no timeout in Axios.

Reply:
We could set a limited number of retries on the axios-retry logic.

Reply:
this timeout is by individual request isn't it? I imagine you'd want the first request to timeout much faster than 60 seconds

Reply:
I don't think so, there is a `shouldResetTimeout` option that defaults to `false`: https://github.com/softonic/axios-retry#options